src/backend/Functions/ProcessWorkflowQueue.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Extensions.Logging;
+using WorkflowManagement;
+
+namespace Functions
+{
+    public static class ProcessWorkflowQueue
+    {
+        //Allows WorkflowManager to be "injected" as needed, such as with a mock action factory when the function is called through tests
+        public static Func<WorkflowManager> GetWorkflowManager { get; set; } = () => new WorkflowManager();
```

Comment:
Is there a reason to have 2 static instances? Also, do they need to be funcs or is WorkflowManager designed to be a static? It would be better to at least follow the service locator plan until we can get DI set up here. That is, created a static class (ie `ServiceLocator`) that has a single static property `WorkFlowManager` that returns it (a single instance or a shared instance depending on what needs to be done).

Reply:
The reason I made WorkflowManager be static is for testability - so that when used within the context of tests, the WorkflowManager is instantiated using mock workflow actions prior to the Azure Function executing.  Since WorkflowManager is static, when the Azure Function runs, a new instance of WorkflowManager isn't re-created with real actions.  I was using the Service Locator pattern before, but since we didn't have DI, I thought this would be a simpler approach for now.  Anyway, I can switch it back to using Service Locator pattern.